iocBoot/ioctiming/runProcServ.sh

```diff
@@ -20,7 +20,7 @@ set -u
 
 # Run run*.sh scripts with procServ
 if [ "${UNIX_SOCKET}" ]; then
-    /usr/bin/procServ -f -n TIMING_${DEVICE_TYPE} -i ^C^D unix:./procserv.sock ./runGenericModule.sh "$@"
+    /usr/bin/procServ -L-  -f -n TIMING_${DEVICE_TYPE} -i ^C^D unix:./procserv.sock ./runGenericModule.sh "$@"
```

Comment:
this shouldn't be `/usr/bin/procServ -L -f -n TIMING_${DEVICE_TYPE} -i ^C^D unix:./procserv.sock ./runGenericModule.sh "$@"` ?

Reply:
No.

`-L` is the flag to enable logging, and as an argument it takes the file to log to, or `-` to specify logging to `stdout`.